README.markdown

```diff
@@ -51,7 +50,7 @@ The inifile module tries hard not to manipulate your file any more than it needs
 
 Use the `ini_subsetting` type:
 
-~~~
+~~~puppet
 JAVA_ARGS="-Xmx192m -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/var/log/pe-puppetdb/puppetdb-oom.hprof "
```

Comment:
This should be broken into two code blocks also... the example is configuring the `-Xmx192m` section to be `-Xmx512m`, but then this line should have 512 not 192.

Also there is an extra space before the "